CHANGELOG.md

```diff
@@ -2,5 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [0.1.0] - 2019-01-14
+### Added
+- Improve HURUmap to use data releases
+- Improve Docker
+- Decouple `hurumap.dashboard`
+- Added new Wazimap logo for dark backgrounds
+- Multiple Chart Qualifiers Support
+
+### Changed
+- Hide Empty sections on the profile page
+- Rename `wazi-logo` to `wazi-logo-white`
+- Replace sourceAFRICA with openAFRICA
```

Comment:
Have the docs/README been updated to show how to use data releases, multiple chart qualifiers and how to hide empty sections at least?